summary-visualiser/templates/scenarios/app_install.html.tmpl

```diff
@@ -2,8 +2,8 @@
 <p>Measure app installation speed. This scenario has two roles:</p>
 
 <ul>
-    <li><code>minimal</code>: Installs a simple app which implements initialisation callbacks but otherwise doesn't contain a lot of code.</li>
-    <li><code>large</code>: Installs a larger app that contains some dependencies and some generated data as padding to make the bundle larger.</li>
+    <li><b><code>minimal</code>: Installs a simple app which implements initialisation callbacks but otherwise doesn't contain a lot of code.</li>
+    <li><b><code>large</code>: Installs a larger app that contains some dependencies and some generated data as padding to make the bundle larger.</li>
```

Comment:
Missing closing b tag

Reply:
huh, and even the Rabbit told me about that and I missed it. Fixed in https://github.com/holochain/wind-tunnel/pull/371/commits/aeef55c17376d40289b914ebb240dd1e17733029